modules/programs/firefox.nix

```
@@ -757,6 +765,19 @@ in {
       }
 
       (mkNoDuplicateAssertion cfg.profiles "profile")
+
+      (let
+        assertProfile = profile:
+          profile.userChrome != ""
+          -> (profile.settings."toolkit.legacyUserProfileCustomizations.stylesheets"
+            != false);
+      in {
+        assertion = any assertProfile (attrValues cfg.profiles);
+        message = ''
+          'userChrome' won't work with
```

```suggestion
          'userChrome' and/or 'userContent' don't work with
```
Unsure on the wording here

---

I added separate assertions with separate messages. This will be more clear for the users.